man/systemd.network.xml

```diff
@@ -247,6 +247,72 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+      <title>[SRIOV] Section Options</title>
```

Comment:
Maybe name the section `[SR-IOV]` (i.e. I see no reason to drop the dash there...

(also please fix the commit msg: SOR-IOV → SR-IOV)